.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Please remove these .idea files, its hard to review having to sift through this rubbish